### ISSUE TITLE
thread 'rustc' panicked at 'index out of bounds: the len is 68 but the index is 68'

### ISSUE THREAD

**Issue by Mromson** [rust-lang/rust#80032]
Unfortunately, I am unable to reproduce this bug. It happened only once so far, the same code compiled a second time without issue...

### Code

```Rust
<unknown>
```

### Meta

`rustc --version --verbose`:
```
rustc 1.48.0 (7eac88abb 2020-11-16)
binary: rustc
commit-hash: 7eac88abb2e57e752f3302f02be5f3ce3d7adfb4
commit-date: 2020-11-16
host: x86_64-pc-windows-msvc
release: 1.48.0
LLVM version: 11.0
```

### Error output

```
$ cargo test -- --nocapture
   Compiling m_proj v0.0.0 (G:\mProj)
thread 'rustc' panicked at 'index out of bounds: the len is 68 but the index is 68', compiler\rustc_middle\src\ty\query\on_disk_cache.rs:706:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace  

error: internal compiler error: unexpected panic

note: the compiler unexpectedly panicked. this is a bug.

note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md

note: rustc 1.48.0 (7eac88abb 2020-11-16) running on x86_64-pc-windows-msvc    

note: compiler flags: -C embed-bitcode=no -C debuginfo=2 -C incremental        

note: some of the compiler flags provided by cargo are hidden

error: could not compile `m_proj`

To learn more, run the command again with --verbose.
```

<details><summary><strong>Backtrace</strong></summary>
<p>

```
<backtrace>
```

</p>
</details>

**Comment by jyn514** [comment]
@Mromson can you provide the code the compiler panicked on? Especially without a backtrace, this isn't really actionable.

**Comment by Mromson** [comment]
@jyn514 I don't think I can provide any meaningful code, as I've been unable to reproduce this issue a second time when I tried to run it again with `RUST_BACKTRACE=full`. All successive build/test attempts succeeded. :/ I already assumed that this bug report probably wasn't actionable, however I figured I'd report it anyways.

**Comment by tgnottingham** [comment]
Based on the line number and version number, this is highly likely to be fixed in 1.49.0 by #76256.

**Comment by jyn514** [comment]
I think it makes sense to close this then - it's unlikely to get reproduction steps and from we can tell it's already been fixed.
